### PR TITLE
Trrack vis

### PR DESCRIPTION
**Pull request by JakeWags**
### Give a longer description of what this PR addresses and why it's needed
Adds Trrack Vis and provenance graph view as well as changes the icons on the element sidebar.

### Are there any additional TODOs before this PR is ready to go?

TODOs:
- [ ] ...

### CODE REVIEW

File: packages/upset/src/components/ElementView/ElementSidebar.tsx [hunk 1]
```diff
@@ -98,17 +100,16 @@ export const ElementSidebar = ({ yOffset }: { yOffset: number }) => {
...
         sx={{
-          width: hide ? 0 : fullWidth ? '100%' : drawerWidth,
+          width: hideElementSidebar ? 0 : fullWidth ? '100%' : drawerWidth,
           flexShrink: 0,
           '& .MuiDrawer-paper': {
             padding: '1em',
-            paddingTop: `${yOffset}px`,
-            width: hide ? 0 : fullWidth ? '100%' : drawerWidth,
-            visibility: hide ? 'hidden' : 'initial',
+            paddingTop: `${yOffset + 5}px`,
```
Comment (JackWilb) [inline packages/upset/src/components/ElementView/ElementSidebar.tsx:107]: Why add 5 here? Can this not just be included in the `yOffset` variable?

File: packages/upset/src/components/ElementView/ElementSidebar.tsx [hunk 2]
```diff
@@ -127,25 +128,36 @@ export const ElementSidebar = ({ yOffset }: { yOffset: number }) => {
...
             onClick={() => {
               setFullWidth(true);
             }}
-            disabled={fullWidth}
           >
-            <ChevronLeftIcon />
-          </IconButton>
+            <OpenInFullIcon />
+          </IconButton>:
           <IconButton
```
Comment (JackWilb) [inline packages/upset/src/components/ElementView/ElementSidebar.tsx:143]:
```suggestion
          </IconButton>
          : <IconButton
```

**Review by JackWilb — COMMENTED** [review]
A couple of question. Waiting on some pictures for approval

File: packages/upset/src/components/ElementView/ElementSidebar.tsx (hunk 1 above)
Comment (JakeWags) [reply packages/upset/src/components/ElementView/ElementSidebar.tsx]: Addressed in commit 152aa75

**Comment by JakeWags** [comment]
# Element sidebar
## Sidebar
image

## Fullscreen
image

# Trrack Vis
image

**Review by JackWilb — APPROVED** [review]

**Pull request merged**